src/app/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react';
+import styles from './Navigation.module.css';
+
+type headerTitleProps = {
+  headerTitle: string;
+};
+
+function Navigation({ headerTitle }: headerTitleProps): JSX.Element {
+  const [navOpen, setNavOpen] = useState(false);
+
+  function handleClick() {
+    setNavOpen(!navOpen);
+  }
+
+  return (
+    <nav className={styles.container}>
+      <h1 className={styles.headerTitle}>{headerTitle}</h1>
+      <ul className={`${styles.navLinks} ${navOpen ? styles.navOpen : ''} `}>
+        <li>
+          <a className={styles.activeLink} href="">
```

Comment:
Does this work? I thought anchor tags aren't allowed to be nested inside lists. Maybe you don't need to create a list here, since you set list-style to none. You could use <span> as an inline-text-element instead.

Reply:
Good point, thank you! Anchor works here fine and I initially put it there to be replaced by Link when I do the routing.